### PR TITLE
COG-1367 CREATE A NEW USER button is disabled in case no search has been performed

### PR DESCRIPTION
**Pull request by sorych**
#### Which User story does this relate to (link)?
https://osi-cwds.atlassian.net/browse/COG-1367

#### Brief feature description
CREATE A NEW USER button is disabled in case no search has been performed

### CODE REVIEW

File: app/javascript/views/searchUserList/SearchUserList.jsx [hunk 1]
```diff
@@ -75,6 +75,20 @@ class SearchUserList extends PureComponent {
...
+
+  isDisabledAddUsrBtn = () => {
+    const { query } = this.props
+    const lastName = query.find(({ field }) => field === 'last_name')
+    const firstName = query.find(({ field }) => field === 'first_name')
+    const officeSearch = query.find(({ field }) => field === 'office_ids')
+    const racfid = query.find(({ field }) => field === 'racfid')
+    const email = query.find(({ field }) => field === 'email')
+    return [lastName, firstName, officeSearch, racfid, email].every(this.isSearchValueAbsent)
+  }
```
Comment (sorych) [inline app/javascript/views/searchUserList/SearchUserList.jsx]: @sunilb0575 @Nasir2712  you're gonna like this ^^ ))

Comment (sunilb0575) [reply app/javascript/views/searchUserList/SearchUserList.jsx]: Nice work, @sorych

File: app/javascript/views/searchUserList/SearchUserList.jsx [hunk 2]
```diff
@@ -75,6 +75,20 @@ class SearchUserList extends PureComponent {
     return lastNameSearch && lastNameSearch.value === lastName
   }
 
+  isSearchValueAbsent = node => {
```
Comment (RamuKammagani-osi) [inline app/javascript/views/searchUserList/SearchUserList.jsx]: Can you add a test case for this function as well?

Comment (sorych) [reply app/javascript/views/searchUserList/SearchUserList.jsx]: @RamuKammagani-osi  I added a test

File: app/javascript/views/searchUserList/SearchUserList.test.js [hunk 3]
```diff
@@ -453,6 +429,83 @@ describe('SearchUserList', () => {
...
+          includeInactive={false}
+          auditEvents={auditEvents}
+          userDetails={details}
+          searchPageTiles={searchPageTiles}
+        />
+      )
+      expect(component.instance().isDisabledAddUsrBtn()).toEqual(false)
+    })
+  })
+
```
Comment (sunilb0575) [inline app/javascript/views/searchUserList/SearchUserList.test.js]: Can you also include another scenario for **office_ids**?

Comment (sorych) [reply app/javascript/views/searchUserList/SearchUserList.test.js]: done

**Pull request merged**